.gitignore

```diff
@@ -22,7 +22,7 @@ node_modules/
 dist/
 
 # Experimental Schemas
-utils/output/*.json
+utils/output/**/*.*
```

Comment:
If you want to ignore the entire outputs, just `utils/output/` is sufficient

Reply:
Didn't see this till after but yea, slight oversight, not a big deal